### PR TITLE
Add view to edit/delete installments

### PR DESCRIPTION
**Pull request by walterbucolo-eb**
Views to edit and delete installment was added. Factory boy was used to test it.

Links and scripts that was in head of each html file created were moved to base.html.

### CODE REVIEW

File: app/templates/app/delete_installment.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+{% load staticfiles %}
+{% load i18n %}
+{% load render_table from django_tables2 %}
+{% block content %}
+<html>
+    <body>
+    <div class='m-5 p-3'>
+        <form method="post">{% csrf_token %}
+        <div class="row justify-content-center m-3">
+                    <div class="card" style="width: 30rem;">
```
Comment (leonardoa-eb) [inline app/templates/app/delete_installment.html]: Could you put this in a css related file? `style="width: 30rem;"` It isn't a good practice https://stackoverflow.com/questions/2612483/whats-so-bad-about-in-line-css

File: app/templates/app/delete_installment.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+                          <li class="m-2"> <b>Upfront projection: </b> {{ object.upfront_projection }}</li>
+                          <li class="m-2"> <b>Recoup amount: </b> {{ object.recoup_amount }}</li>
+                          <li class="m-2"> <b>Status: </b> {{ object.status }}</li>
+                          <li class="m-2"> <b>GTF: </b> {{ object.gtf }}</li>
+                          <li class="m-2"> <b>GTS: </b> {{ object.gts }}</li>
+                          <div class="row m-4">
+                              <div class="col justify-content-md-center">
+                                  <a href="{% url 'installments-create' object.contract_id %}" role="button" class="btn btn-outline-secondary"><i class="far fa-arrow-alt-circle-left"></i>  Back</a>
+                              </div>
+                              <form method="post">{% csrf_token %}
```
Comment (leonardoa-eb) [inline app/templates/app/delete_installment.html]: Why you need this `<form>` , what's about of line 12?

File: app/templates/app/delete_installment.html [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+                                  <div class="col row justify-content-md-end"><button type="submit" class="btn btn-danger">Delete</button></div>
+                              </form>
+                          </div>
+                        </p>
+                      </div >
```
Comment (leonardoa-eb) [inline app/templates/app/delete_installment.html]: nit: `</div>` without white-space

File: app/templates/app/update_installment.html [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+                        <input type='submit' class="btn btn-primary" value='Save' />
+                    </div>
+                </div>
+            </div>
+        </form>
+    <script>
+        $( function() {
+            $( "#datepicker_maximum_payment_date" ).datepicker();
+            $( "#datepicker_payment_date" ).datepicker();
+        } );
```
Comment (leonardoa-eb) [inline app/templates/app/update_installment.html]: Could you move this to a related js file?

File: app/templates/base.html [hunk 5]
```diff
@@ -6,6 +6,15 @@
 <link rel="stylesheet" type="text/css" href="{% static 'css\css.css' %}">
 <html lang="en">
 <head>
+  <link rel="stylesheet" href="../../static/css/uf_table.css">
+  <link href="https://maxcdn.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" rel="stylesheet" id="bootstrap-css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.8.0/css/all.css">
+  <link rel="stylesheet" href="http://code.jquery.com/ui/1.10.1/themes/base/jquery-ui.css" />
```
Comment (leonardoa-eb) [inline app/templates/base.html]: Could do you opt-in for `.min.` versions for these assets?

File: app/views.py [hunk 6]
```diff
@@ -14,6 +14,7 @@
     ListView,
     UpdateView,
     TemplateView,
+    DeleteView,
```
Comment (leonardoa-eb) [inline app/views.py]: Pls sort this imports

File: app/templates/app/delete_installment.html (hunk 2 above)
Comment (walterbucolo-eb) [reply app/templates/app/delete_installment.html]: I don't need it, I made a mistake.

**Pull request merged**